alodataset/coco_base_dataset.py

```diff
@@ -78,10 +78,17 @@ def __init__(
             return
         else:
             assert img_folder is not None, "When sample = False, img_folder must be given."
-            assert ann_file is not None, "When sample = False, ann_file must be given."
+            assert ann_file is not None or  "test"  in img_folder, "When sample = False and the test split is not used, ann_file must be given."
```

Comment:
undesirable double space

Reply:
I actually commited this file by mistake, it was already merged in #247